src/sentry/tasks/summaries/utils.py

```diff
@@ -413,10 +413,22 @@ def organization_project_issue_substatus_summaries(ctx: OrganizationReportContex
         project_ctx.total_substatus_count += item["total"]
 
 
-def check_if_project_is_empty(project_ctx: ProjectContext) -> bool:
+def check_if_project_is_empty(project_ctx: ProjectContext, daily: bool) -> bool:
     """
-    Check if this project has any content we could show in an email.
+    Check if this project has any content we could show in a notification.
     """
+    if daily:
+        return (
+            not project_ctx.key_errors
```

Comment:
@ceorourke I would push this logic to `ProjectContext` so this function doesn't have to know all the fields that we are checking